### ISSUE TITLE
Accessibility bug - Declarative table column is not announced with screen reader

### ISSUE THREAD

**Issue by bharathpalaksha** [microsoft/azuredatastudio#25443]
- Azure Data Studio Version: 1.47.0
- OS Version: Windows 11, mac, linux

This accessibility bug is on the extension and we use Azure data studio components, here DeclarativeTable's column header is not getting announced by screen reader

Steps to Reproduce:

https://github.com/microsoft/azuredatastudio/assets/342603/0003a2f6-a342-4618-a216-f26d7491fbed

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by Samhitha66** [comment]
#A11yMAS; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio;
